conf/Makefile.chibios

```diff
@@ -131,49 +168,40 @@ endif
 # Project, sources and paths
 #
 
-# Define project name here
-PROJECT = ap
+# Project name defined in board makefile
 
 # Imported source files and paths
-
-CHIBIOS = $(CHIBIOS_EXT)
-OPENCM3 = $(OPENCM3_EXT)
-OPENCM3_INC = $(OPENCM3)/include
-OPENCM3_LIB =  $(OPENCM3)/lib/
-
-include $(CHIBIOS_BOARD_DIR)/board.mk
-include $(CHIBIOS)/os/hal/platforms/STM32F4xx/platform.mk
+# Startup files.
+include $(CHIBIOS)/os/common/ports/ARMCMx/compilers/GCC/mk/startup_stm32f4xx.mk
```

Comment:
Oh I see, it is already there - just the first include is weird (the startup script).